docs/managed_2gp.rst

```diff
@@ -0,0 +1,162 @@
+are deployed when CumulusCI creates a scratch org, and are installed in the packaging org when CumulusCI creates 1GP package versions.However, second-generation packaging does not have a packaging org, and does not allow interactive access to the build org. 
+
+CumulusCI offers two modes of handling unpackaged metadata owned by dependencies when building a second-generation package. 
+
+The default behavior is to ignore unpackaged metadata. If unpackaged metadata is intended to satisfy install-time dependencies
+of packages, this requires that those dependencies be met in other ways, such as by configuring the scratch org definition. For
+examples of how to satisfy the install-time dependencies for NPSP and EDA without using unpackaged metadata, see :doc:`Extending NPSP and EDA with Second-Generation Packaging <npsp_eda_2gp>`.
+
+The other option is to have CumulusCI automatically create Unlocked Packages containing unpackaged metadata from dependency projects.
+For example, if your project depended on the repository ``Food-Bank``, which contained the unpackaged metadata directories
+
+* ``unpackaged/pre/record_types``
+* ``unpackaged/pre/setup``
+
+CumulusCI would automatically, while uploading a version of your package, upload Unlocked Package versions containing the current
+content of those unpackaged directories.
+
+The Unlocked Package route is generally suitable for testing only, where it may be convenient when working with complex legacy
+projects that include lots of unpackaged metadata. However, it's generally *not* suitable for use when building production packages,
+because your packages would have to be distributed along with those Unlocked Packages. For this reason, this behavior is off by default.
+If you would like to use it, configure your ``cumulusci.yml`` to set the option ``create_unlocked_dependency_packages`` on the
+``create_package_version`` task.
+
+Test a Beta Version
+-------------------
+
+The ``ci_beta`` flow installs the latest beta version of the project in a scratch org, and runs Apex tests against it.
+
+.. code-block:: console
+
+    $ cci flow run ci_beta --org beta 
+
+This flow is intended to be run whenever a beta release is created.       
+
+
+Promote a Production Version
+----------------------------
+
+To be installed in a production org, an 2GP package version must be `promoted <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_pkg_ver_promote.htm>`_ to mark it as released.
+
+To promote a production release of your managed package project:
+
+.. code-block::
+
+    $ cci flow run release_2gp_production --org packaging 
+
+Unlike first-generation packages, promoting a second-generation package doesn't upload a new version. Instead, it promotes the most recent beta version (found in the project's GitHub releases) to production status. Then, CumulusCI creates a new, production GitHub
+release, and aggregates release notes for that release.
```

Comment:
How are the beta and production github releases distinguished, since the version number is the same?

Reply:
By the tag prefix, and by the latter being marked as a production release in GitHub.